WordPress/Sniffs/WP/AlternativeFunctionsSniff.php

```diff
@@ -126,11 +126,29 @@ public function getGroups() {
 				),
 			),
 
+			'unlink' => array(
+				'type'      => 'warning',
+				'message'   => '%s() is discouraged. Use wp_delete_file() for remote URLs instead.',
+				'since'     => 'x.x.x',
+				'functions' => array(
+					'unlink',
+				),
+			),
+
 			'file_system_read' => array(
```

Comment:
```suggestion
			'file_system_operations' => array(
```

The `file_system_read` name is inaccurate. It is like this for historic reasons and changing it would be a - albeit minor - breaking change. Having said that, we are now at `3.0.0`, so if we want to fix the name, now would be the time.

Reply:
Updated